src/components/SlateEditor/EditorFooter.tsx

```diff
@@ -152,7 +162,7 @@ const EditorFooter: React.FC<Props> = ({
           </FooterLinkButton>
```

Comment:
skal vi ha preview på concepts? isåfall hva blir linken da? for /preview/ er reservert til drafts, /preview/concepts/:id ?

Reply:
Concepts forhåndsvises i artikkelen. Vi treng ikkje ha en egen forhåndsvisningsside her.